.vsts-ci.yml

```diff
@@ -1,5 +1,22 @@
 # Responsible for producing the signed product build for insertion into the Visual Studio repository
 
+pr: none
+# Branches that trigger a build on commit
+trigger:
+  branches:
+    include:
+    - master
```

Comment:
Do we have to list these branches?

Reply:
If we don't then we will need to override the set of branches on the build defintion.  I think its better to list these out in the file instead of having to wrestle with a web UI